spec/models/importers/all_organisations_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe Importers::AllOrganisations do
+  describe '#import_all' do
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/38/commits/f68ab65364b9529a8166dff70abd82f882bc3edc